### DISCUSSION TITLE
BMS Connection

### DISCUSSION THREAD

**Discussion by MaartenGoris** [ArminJo/JK-BMSToPylontechCAN#8] [General]
Hi,

Can you show a picure of the connection to the bms?

I am interested in this project to test is wit a Growatt SPH6000

Nice work!

**Comment by ArminJo** [comment]
It is on the pictures.

**Reply by MaartenGoris** [reply]
I see, in yourre bms the connection is at the side.
I have a front connection. 
![WhatsApp Image 2023-09-03 at 12 37 59](https://github.com/ArminJo/JK-BMSToPylontechCAN/assets/42836064/c421634c-7008-406d-b60c-53b3cf94dfd8)

I supose i need the CAN/RS485 connection to do this.

Any idea of the PIN layout?

**Reply by ArminJo** [reply]
The pin layout is decribed [here ](https://github.com/ArminJo/JK-BMSToPylontechCAN#connection-schematic)

**Reply by MaartenGoris** [reply]
Indeed i saw that already, but i can assume bottom is left pin.

**Reply by MaartenGoris** [reply]
Hi,

It is hard to find a connector for the 3 Pin, i als saw you can use the 4Pin micro jst.
But the label says GPS is this also just TX an RX?
Why is the connection pin for this connection RX to D4 on Uno and not TX?

**Reply by ArminJo** [reply]
> Why is the connection pin for this connection RX to D4 on Uno and not TX?

Good Question! -> I added some documentation 😀 .
